.github/workflows/tests.yml

```diff
@@ -19,9 +19,9 @@ jobs:
       BUNDLE_GEMFILE: ${{matrix.gemfile}}
     strategy:
       matrix:
-        ruby: ['3.0', '2.7']
+        ruby: ['3.1', '2.7']
```

Comment:
Fix this CI failure: https://github.com/SenteraLLC/activerecord-mysql2rgeo-adapter/actions/runs/10603271591/job/29387206194?pr=5